### PR TITLE
Document iptables interaction with Openshift

### PR DESCRIPTION
**Pull request by ahardin-rh**
Continuation of https://github.com/openshift/openshift-docs/pull/3000

### CODE REVIEW

**Comment by ahardin-rh** [comment]
Preview Build:
http://file.rdu.redhat.com/~ahardin/11102016/PR3000-pecameron/admin_guide/iptables.html

@adellape @bfallonf PTAL with special attention given to Docker/container language per latest guidelines

File: admin_guide/iptables.adoc [hunk 1]
```diff
@@ -11,18 +11,97 @@
...
+software that manage local firewall policies that rely on the kernel iptables
+configuration for proper network operation. In addition, the iptables
+configuration of all nodes in the cluster must be correct for networking to
+work.
 
+All components independently work with iptables without knowledge of how other
+components are using them. This makes it very easy for one component to break
+another component's configuration. Further, {product-title} and the Docker service
+assume that iptables remains set up exactly as they have set it up. They may not
+detect changes introduced by other components and if they do there maybe some
```
Comment (bfallonf) [inline admin_guide/iptables.adoc]: s/maybe/may be

File: admin_guide/iptables.adoc [hunk 2]
```diff
@@ -11,18 +11,97 @@
...
+----
+# systemctl disable iptables.service
+# systemctl mask iptables.service
+----
+
+If you need to run `iptables.service`, keep a limited configuration in the
+configuration file and rely on {product-title} and Docker to install their
+needed rules.
+
+The `iptables.service` configuration is loaded from:
```
Comment (bfallonf) [inline admin_guide/iptables.adoc]: If `iptables.service` is a service, should it be with asterisks? 

> *iptables.service*

**Comment by bfallonf** [comment]
@ahardin-rh A few comments, but otherwise LGTM!

**Comment by knobunc** [comment]
@pecameron PTAL

@ahardin-rh will this also go into 3.4?

**Comment by ahardin-rh** [comment]
@knobunc Thanks! I have it marked as Next Release, which means it will go out in our next weekly release (which I am scheduled to manage this Monday, Nov 14). If this is for 3.4 only or if it needs to wait until the 3.4 release, I can adjust the labels and milestone accordingly. Just let me know. Thanks again!

**Comment by knobunc** [comment]
@ahardin-rh no, it should be 3.3 and 3.4... I just wasn't sure how the 3.4-ness was labeled.  Thanks!

**Comment by pecameron** [comment]
@ahardin-rh @knobunc  LGTM

**Comment by ahardin-rh** [comment]
Thanks all! Addressed comments and merging.

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../admin_guide/iptables.adoc#admin-guide-iptables[iptables]
|New topic on how {product-title} and iptables interact.
%
|xref:../admin_guide/sdn_troubleshooting.adoc#admin-guide-sdn-troubleshooting[Troubleshooting OpenShift SDN]
|Removed references to`firewalld`, as the installer does not currently support it.
%

**Pull request merged**